geo-types/CHANGES.md

```diff
@@ -1,20 +1,20 @@
 # Changes
 
-## Unreleased
+## 0.7.2
 
 * Implement `RelativeEq` and `AbsDiffEq` for fuzzy comparison of remaining Geometry Types
   * <https://github.com/georust/geo/pull/628>
 * Implement `From<Line>` for `LineString`
   * <https://github.com/georust/geo/pull/634>
+* Add optional `arbitrary` feature for integration with the [arbitrary](https://github.com/rust-fuzz/arbitrary) crate
+  * <https://github.com/georust/geo/pull/622>
 
 ## 0.7.1
 
 * Implement `Default` on `Coordinate` and `Point` structs (defaults to `(x: 0, y: 0)`)
   * <https://github.com/georust/geo/pull/616>
 * Add specific details about conversion failures in the newly public `geo_types::Error`
   * <https://github.com/georust/geo/pull/614>
-* Add optional `arbitrary` feature for integration with the [arbitrary](https://github.com/rust-fuzz/arbitrary) crate
-  * <https://github.com/georust/geo/pull/622>
```

Comment:
I think this was in the wrong spot - although the `arbitrary` feature was merged in time for the previous *geo* release (0.17.1), the most recent *geo-types* release (0.7.1) predates that.

/cc @frewsxcv